apps/web/src/app/settings/pull-request-settings/size/page.tsx

```
@@ -0,0 +1,100 @@
+  };
+
+  return (
+    <DrawerScrollable
+      {...drawerProps}
+      title="Pull Request Size"
+      actions={
+        <>
+          <Button type="submit" loading={isPending} disabled={!isFormValid}>
+            Save
+          </Button>
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)